version.json

```diff
@@ -1,8 +1,8 @@
 {
   "$schema": "https://raw.githubusercontent.com/dotnet/Nerdbank.GitVersioning/main/src/NerdBank.GitVersioning/version.schema.json",
-  "version": "17.7",
+  "version": "17.8",
```

Comment:
Can you also drop buildNumberOffset?

Reply:
Drop as in remove, or as in lower? Can you propose a suggestion on the PR here?

Reply:
As drop as in remove, added a suggestion.